webapp/src/onboarding.ts

```diff
@@ -29,6 +29,9 @@ function getTargetMap(target: string): querySelector {
         "download": {
             targetQuery: "#downloadArea",
         },
+        "play button" : {
```

Comment:
Will there be any problems with having a space in this name? Not sure how these are parsed.

Reply:
^ I had the same thought but the ones just above also have spaces and are working fine~

Reply:
Yea good question, it works fine with the space.